hatohol.spec.in

```diff
@@ -149,6 +149,14 @@ Requires: hatohol-hap2-common = %{version}
 %description hap2-nagios-ndoutils
 Nagios ndoutils plugin for hatohol arm plugins version 2.0.
 
+%package hap2-nagios-livestatus
+Summary: Nagios livestatus plugins of HAPI2.0
+Group: Applications/System
+Requires: hatohol-hap2-common = %{version}
```

Comment:
hap2-plugin can work on host that does not exist Nagios and check-mk-livestatus module.
So, we do not need to add check-mk-livestatus.
But python-mk-livestatus module is required. I install it in %post section.